react/features/external-api/middleware.js

```diff
@@ -39,7 +39,7 @@ MiddlewareRegistry.register(store => next => action => {
 
         const result = next(action);
 
-        if (participant.loadableAvatarUrl !== loadableAvatarUrl) {
+        if (participant && (participant.loadableAvatarUrl !== loadableAvatarUrl)) {
             APP.API.notifyAvatarChanged(
                 id,
                 loadableAvatarUrl
```

Comment:
Below, the case for CONFERENCE_FAILED, is what's screwing up electron. Notice it's notifying of the current avatar url, which might be abotars.